docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,134 @@
-Ubuntu specific instructions
\ No newline at end of file
+# Installing LORIS In-Depth
+
+This guide assumes an installation on an Ubuntu server. 
+
+If you are using CentOS, please visit the CentOS installation guide for 
+instructions on installation.
+
+We also have instructions for how to set up LORIS for macOS. However, only
+ Ubuntu and CentOS are officially supported by the team so we may be unable to
+ help you with issues arising on other systems.
+
+When you've completed this guide, you should be able to load and log into LORIS in your browser. Further setup to customize LORIS for your project will be required after that. Please visit
+[the Setup page](https://github.com/aces/Loris/wiki/Setup) in order to complete the setup for LORIS.
+
+## Prequisities
+
+### Server Stack
+
+LORIS requires a LAMP stack in order to run, specifically:
+* Apache 2.4
+* MySQL 5.7 (or MariaDB 10.3) (or higher)
+* PHP 7.3 (or higher)
+
+Additionally, LORIS requires the following package managers:
+* npm 8.0 (or higher)
```

Comment:
@johnsaigle I think this should say Node 8.0
It looks like npm 's release numbers are not that high. 
(reported via Blake Fitch in Tuebingen)